### PR TITLE
Revision terminology.schema.json

### PR DESCRIPTION
**Pull request by UlrikeS91**
Revision of schema based on summary in #31.

### CODE REVIEW

**Comment by UlrikeS91** [comment]
Ignore this PR until the re-revision. During our meeting today, we decided to remove all version related properties.

**Review by lzehl — CHANGES_REQUESTED** [review]

File: v0.1/core/terminology.schema.json
Comment (UlrikeS91) [reply v0.1/core/terminology.schema.json:4]: Hm...Don't know how this happened :D I'll fix it.

**Comment by UlrikeS91** [comment]
@lzehl I fixed the typo. It can be merged now, right?

**Comment by lzehl** [comment]
Please correct also already the schema name (`atlasTerminology`) and definitions file in this PR. 
I forgot about it in the first review.

**Comment by UlrikeS91** [comment]
> * `terminology.schema.json` should be changed to `atlasTerminology.schema.json`
> * `definitions.json` and `brainAtlasVersion.schema.json` need to be correspondingly adopted

Okay, saw this comment too late. See #31, but instead of in a new PR, I'll do it here.

**Comment by lzehl** [comment]
so you do it here now? :joy:

**Comment by UlrikeS91** [comment]
> so you do it here now? 😂

Yes :D

**Comment by lzehl** [comment]
@UlrikeS91 based on #26 you can remove terminology from anatomicalEntity.scheam.json

**Comment by UlrikeS91** [comment]
@lzehl please review changes. Updated several files according to what I summarised in #31 (definitions file, schema refs, schema name itself and everything within the schema itself.

**Comment by UlrikeS91** [comment]
> @UlrikeS91 based on #26 you can remove terminology from anatomicalEntity.scheam.json

I'll update anatomicalEntity.schema.json last/wait until you merged terminology changes into the master to avoid conflicts.

**Comment by lzehl** [comment]
Okay! :+1:

**Pull request merged**
